osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/IdentificationNumber.java

```diff
@@ -63,12 +63,10 @@ public static IdentificationNumber fromNumericalRepresentation(
     return new IdentificationNumber(String.format("%08d", identificationInNumericalRepresentation));
   }
 
-  private static Long toBcdRepresentationAsLong(
-      final String identificationNumberInTextualRepresentation) {
+  private Long toBcdRepresentationAsLong(final String identificationNumberInTextualRepresentation) {
```

Comment:
Why remove `static` while still getting the value from the parameter? Suggestion: remove the parameter and use `this.textualRepresentation`.